### PR TITLE
#1308 - STR function missing from SparqlFunctions

### PR DESCRIPTION
**Pull request by reckart**
This PR addresses GitHub issue: #1308.

Briefly describe the changes proposed in this PR:

- Added STR function

### CODE REVIEW

File: sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/SparqlFunction.java [hunk 1]
```diff
@@ -54,6 +54,7 @@
 	SHA256("SHA256"),
 	SHA384("SHA384"),
 	SHA512("SHA512"),
+    STR("STR"),
```
Comment (jeenbroekstra) [inline sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/SparqlFunction.java]: Minor thing but can you please fix the indentation? We standardize on tabs-only.

**Comment by reckart** [comment]
Done

File: sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/SparqlFunction.java (hunk 1 above)
Comment (reckart) [reply sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/SparqlFunction.java]: Done

**Comment by reckart** [comment]
Would be good if RDF4J came with Checkstyle such that the IDE can show such problems immediately. 

Given that tabs resolve to different sizes depending on where one views the file, I'd personally also vote for a spaces policy instead.

**Comment by jeenbroekstra** [comment]
@reckart we've got code formatting settings for Eclipse (see the contributor guidelines), but having something independent from the IDE, like checkstyle, is under discussion.

And I'm not getting involved in a tabs vs spaces war except to point out that the official Eclipse code guidelines standardize on tabs 🙂

**Pull request merged**